src/coreclr/src/jit/indirectcalltransformer.cpp

```diff
@@ -439,7 +439,14 @@ class IndirectCallTransformer
                 fatCall->gtCallArgs = compiler->gtPrependNewCallArg(hiddenArgument, fatCall->gtCallArgs);
             }
 #else
-            AddArgumentToTail(fatCall->gtCallArgs, hiddenArgument);
+            if (fatCall->gtCallArgs == nullptr)
```

Comment:
Shouldn't this check actually be in `AddArgumentToTail`?

Reply:
We would have to change the signature of the method and either pass `fatCall->gtCallArgs` by reference or have it return a `Use*` that we can assign. Doable, but I already merged this and I really dislike dealing with the JIT code formatter (which I'm sure would moan about something).

Reply:
@MichalStrehovsky if you use jitutils then jit code formatting is pretty simple

* clone [jitutils repo](https://github.com/dotnet/jitutils)
* run `bootstrap.cmd` from repo root
* cd to your runtime repo
* run `jit-format --fix --untidy --runtime src\coreclr`

Reply:
@AndyAyersMS thanks! I'm bookmarking your comment